### PR TITLE
Profile update for the `gpus` partition of JUWELS.

### PR DESCRIPTION
**Pull request by Anton-Le**
Update of the PIConGPU profile for the `gpus` partiton of the JUWELS system.

### CODE REVIEW

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example [hunk 1]
```diff
@@ -23,52 +22,46 @@ jutil env activate -p $proj
...
 
 
-export LD_LIBRARY_PATH=$EBROOTICU/lib:$LD_LIBRARY_PATH
-
 # Environment #################################################################
 #
-#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
 
-export PICSRC=$HOME/src/picongpu
+export PICSRC=$HOME/shared/src/picongpu #modified
```
Comment (PrometheusPi) [inline etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:64]: why this comment ?

Comment (Anton-Le) [reply etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:64]: My bad. Failed to remove in clean-up.

**Comment by psychocoderHPC** [comment]
please rebase against the dev branch that we can merge your changes

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example (hunk 1 above)
Comment (psychocoderHPC) [reply etc/picongpu/juwels-jsc/gpus_picongpu.profile.example]: why do you change the path to home shared? Is this a symlink to a projectfolder?

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example [hunk 2]
```diff
@@ -23,52 +22,45 @@ jutil env activate -p $proj
...
 
 
-export LD_LIBRARY_PATH=$EBROOTICU/lib:$LD_LIBRARY_PATH
-
 # Environment #################################################################
 #
-#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
 
-export PICSRC=$HOME/src/picongpu
+export PICSRC=$HOME/shared/src/picongpu
```
Comment (PrometheusPi) [inline etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:63]: This is an example path - why did you change it?

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example [hunk 3]
```diff
@@ -23,52 +22,45 @@ jutil env activate -p $proj
...
+PARTITION_LIB=$PROJECT/lib
+
+
+LIBSPLASH_VER=1.7.0
+PNGWRITER_VER=0.7.0
+BLOSC_VER=1.19.0
+ADIOS_VER=1.13.1
+
+LIBSPLASH_ROOT=$PARTITION_LIB/LIBSPLASH/$LIBSPLASH_VER
+PNGWRITER_ROOT=$PARTITION_LIB/PNGWRITER/$PNGWRITER_VER
```
Comment (PrometheusPi) [inline etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:48]: Did you really name these libraries in upper case letters and not in the common camel-case?

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example [hunk 4]
```diff
@@ -23,52 +22,45 @@ jutil env activate -p $proj
...
-
-HDF5_ROOT=$PARTITION_LIB/hdf5
-export PATH=$HDF5_ROOT/bin:$PATH
-export CMAKE_PREFIX_PATH=$HDF5_ROOT:$CMAKE_PREFIX_PATH
-export LD_LIBRARY_PATH=$HDF5_ROOT/lib:$LD_LIBRARY_PATH
-
-LIBSPLASH_ROOT=$PARTITION_LIB/libSplash
-PNGWRITER_ROOT=$PARTITION_LIB/pngwriter
+# Manually installed libraries are stored in PARTITION_LIB
+PARTITION_LIB=$PROJECT/lib
```
Comment (PrometheusPi) [inline etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:39]: Is `$PROJECT` a system variable?

**Comment by psychocoderHPC** [comment]
Can we please finish this PR.

**Comment by psychocoderHPC** [comment]
Please rebase this PR after applying all changes to pass the CI.

**Comment by psychocoderHPC** [comment]
@Anton-Le Could you please finish this PR an other user requires this PR.

**Comment by psychocoderHPC** [comment]
@Anton-Le Ping

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example (hunk 2 above)
Comment (psychocoderHPC) [reply etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:63]: This change should be removed to be consistent with our documentation

File: etc/picongpu/juwels-jsc/gpus_picongpu.profile.example (hunk 3 above)
Comment (Anton-Le) [reply etc/picongpu/juwels-jsc/gpus_picongpu.profile.example:48]: Yes. Convention I adapted from the module system at MareNostrum 4.

**Comment by psychocoderHPC** [comment]
closed because this PR is now moved to #3359
